src/extensions/common/getRecordingInfo.ts

```diff
@@ -0,0 +1,81 @@
+import { useContext, useRef, useState } from "react";
```

Comment:
This file should be renamed. It contains `useRecordingInfo`, `useSortingInfo`, `useRecordingInfos`, and `useSortingInfos`. What should it be called? Or do we need two separate files?

Reply:
submitted an issue for this.